integration/test_aggregate_autoload.py

```
@@ -0,0 +1,101 @@
+        )
+        result = sorted(rows(reply), key=lambda r: r["cat"])
+        assert result == [
+            {"cat": b"a", "cnt": b"2", "scaled": b"20"},
+            {"cat": b"b", "cnt": b"1", "scaled": b"10"},
+        ]
+
+    def test_unknown_field_still_rejected(self):
+        """Auto-loading must not turn a genuinely unknown field into a silent
+        Nil -- it is not a declared attribute, so the query still fails."""
+        client = self._client()
+        with pytest.raises(Exception):
```

Ruff (0.16.2)

[warning] Do not assert blind exception: `Exception`

(B017)